docs/v1.0/sinks/datadog.md

```diff
@@ -22,6 +22,15 @@ Only one of the methods can be used at a time.
 If `dogstatsd-host` option is set, DogStatsD is used for sending metrics, otherwise `api` keys must be provided.
 If the `doststatsd-host` doesn't include the port, `8125` is used as the default port.
 
+## Counter Metrics
+Previously, blip treated most counter metrics as cumulative counters during collection, except for those that were truncated after collection. Although certain metrics platforms support cumulative counters as a metric type, Datadog only supports delta as counter values.
+
```

Comment:
I'm not sure this is strictly true. It supports cumulative counters, but it doesn't really support treating them as a rate though, whereas other platforms like SignalFX could/would.

Reply:
If you send cumulative counter values to Datadog, even though you can do some calculations with the data (which are not dependant on metric type like `.rate(query)`, sending cumulative counter values datadog confuses their system and the in-app modifiers (`.as_count()` and `.as_rate()`) doesn't work as expected, they are enforced in their UIs (you can remove them from raw query editor but as soon as change something else from the visual query editor they come back).

One of their support rep told me if we wanted to keep the cumulative counter, we would be better off changing the type to `Guage` so their UI won't try to enforce methods which only works with Delta counters.

With all those issues, I think their support for Cumulative counters is confusing _at best_.